### PR TITLE
Add new console search functionality

### PR DESCRIPTION
**Pull request by jprcoliveira**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by jprcoliveira** [comment]
@urbanisierung here's the draft of the entry for the docs. Feedback welcome :)

**Comment by urbanisierung** [comment]
> @urbanisierung here's the draft of the entry for the docs. Feedback welcome :)

Hey @jprcoliveira, did you change more than this?

image

**Comment by jprcoliveira** [comment]
@urbanisierung commit is now in. Clearly, I should use my local codespace and drop my browser-based ones :)

File: docs/components/console/manage-organization/advanced-search.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+This search functionality allows users to:
+
+- Navigate between apps, actions, and tasks (e.g. go to Modeler, invite users, API clients, etc.)
+- Locate Camunda project assets (e.g. BPMN, DMN models, and clusters).
+- Find all entries in the Camunda forum, documentation, and public GitHub issues.
+
+## Open the search bar
+
+Press `ctrl`+`k`, `⌘`+`k`, or click the magnifier in the top navigation bar to open the search bar.
```
Comment (christinaausley) [inline docs/components/console/manage-organization/advanced-search.md:17]: This is non-blocking, but I assume each command is specific to operating system? If so, I would likely note those in parentheses.

**Review by christinaausley — APPROVED** [review]
Technical review in latest commits, and one non-blocking comment to resolve in a separate mini PR perhaps.

**Pull request merged**